ts/features/bonusVacanze/store/actions/bonusVacanze.ts

```diff
@@ -39,12 +39,18 @@ export const loadBonusVacanzeFromId = createAsyncAction(
   "BONUS_LOAD_FROM_ID_REQUEST",
   "BONUS_LOAD_FROM_ID_SUCCESS",
   "BONUS_LOAD_FROM_ID_FAILURE"
-)<string, BonusActivationWithQrCode, Error>();
+)<string, BonusActivationWithQrCode, { error: Error; id: string }>();
 
 export const startLoadBonusFromIdPolling = createStandardAction(
   "BONUS_FROM_ID_START"
 )<string>();
 
+export const loadAllBonusActivations = createAsyncAction(
+  "BONUS_LOAD_ALL_ACTIVATION_REQUEST",
+  "BONUS_LOAD_ALL_ACTIVATION_SUCCESS",
+  "BONUS_LOAD_ALL_ACTIVATION_FAILURE"
+)<void, void, Error>();
```

Comment:
atm only `loadAllBonusActivations.request` is used. Maybe if the other events are not expected to be used can be replaced with a standardAction?

Reply:
@fabriziofff 
It was born like a standard action but in that way I need another action to dispatch when an error occurred (they could be precious on troubleshooting with Mixpanel)
At this time It uses request and failure